InventoryInStorePickupSales/etc/extension_attributes.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Api/etc/extension_attributes.xsd">
+    <extension_attributes for="Magento\Sales\Api\Data\ShipmentInterface">
```

Comment:
Do we need an extension attribute for this interface? I can't find any usage in code.

Reply:
@sidolov, it's used here https://github.com/magento/inventory/pull/2751/files#diff-c7e06eea57ffeac27d7ea0156557a70cR47 To prevent shipment creation via API in case order has delivery method 'in store pick up'. It's not the best solution, but at the moment I have no better ideas to prevent shipment creation, as our notify order for pickup service uses the same shipment creation service as magento does.

Reply:
@sidolov, removed in 7f7877e46aff17efc8041c80f5360b7d19cbf7ff